scripts/templates/field_details.j2

```diff
@@ -136,6 +136,11 @@ Note also that the `{{ fieldset['name'] }}` fields are not expected to be used d
 | {{ entry['short'] }}
 {%- endif %}
 
+{% if entry['normalize'] and 'array' in entry['normalize'] -%}
+
+Note: this reuse should contain an array of <fieldset> objects.
```

Comment:
Is `<fieldset>` a placeholder? Right now this will generate including the brackets:

<img width="715" alt="Screen Shot 2022-03-14 at 5 27 15 PM" src="https://user-images.githubusercontent.com/7226265/158271249-22d3e481-7d1f-4712-8e88-c5a3249c5b95.png">

If you want, you can have the template render the field set name:

```
Note: this reuse should contain an array of {{ entry['name'] }} field set objects.
```